rpm/inicpp.spec

```diff
@@ -0,0 +1,48 @@
+Name: inicpp
+Version: 1.0.0
+Release: 1%{?dist}
+Summary: C++ parser of INI files.
+License: MIT
+BuildRequires: cmake, make, (gcc-c++ or clang)
+URL: https://github.com/SemaiCZE/%{name}
+Patch0: %{name}-cmake.patch
```

Comment:
Can you please introduce a new variable into the top-level `CMakeLists.txt` that can optionally disable looking for `vendor/googletest` directory instead of this patch? Then you can just call something like `cmake -DNO_GOOGLE_TEST=true ..` later in `%build` section.

Reply:
This and other changes suggested changes essentially mean building against the master branch instead of the v1.0.0 tag.  Pushed changes to CMakeLists.txt in master and removed the patch from the rpm subdirectory.  The NO_GOOGLE_TEST option defaults to OFF (i.e. the google test code section will still be processed).  The spec file sets -DNO_GOOGLE_TEST=ON as you suggested.